### ISSUE TITLE
Periodogram normalization doesn't work for custom frequency ranges

### ISSUE THREAD

**Issue by keatonb** [KeplerGO/lightkurve#490]
#### Problem description
The `.to_periodogram()` function allows the periodogram to be calculated across a custom frequency range, but the way the psd normalization is implemented, the units returned are only correct if the periodogram is computed between 0 and the Nyquist.  This is a problem when passing any of the min_frequency, max_frequency, min_period, max_period, or nyquist_factor kwargs through to the astropy periodogram function.  

#### Example

```python
#Min example
import numpy as np
import matplotlib.pyplot as plt
import lightkurve as lk

time = np.arange(1,100,0.1)
flux = 1. + 0.1 * np.sin(2.*np.pi*time) + 0.2*np.random.randn(time.size)

lc = lk.LightCurve(time=time,flux=flux)
fullper = lc.to_periodogram(oversample_factor = 5)
partper = lc.to_periodogram(min_frequency= 0.5, max_frequency= 1.5, oversample_factor = 5)

#these should be the same in the range of frequency overlap
plt.plot(fullper.frequency,fullper.power)
plt.plot(partper.frequency,partper.power,zorder=-1)

plt.title('psd normalization breaks for specified freq ranges')
plt.xlabel('frequency')
plt.ylabel('psd')
plt.tight_layout()
plt.savefig('normprobs.png')
plt.show() 
```
![normprobs](https://user-images.githubusercontent.com/6413923/55485607-1062be80-562b-11e9-9414-22c9db10c03c.png)

#### Expected behavior

These should be the same thing, just computed over different frequency ranges.  However, the power is completely different due to the way the power spectrum is normalized.  Rather than computing the spectrum directly in the desired units such that Parseval's theorem is naturally obeyed, the spectrum is rescaled from arbitrary units to try to enforce Parseval's theorem post hoc.  This is a hack that works only when the power spectrum is computed from 0 to the Nyquist frequency, and importantly only in the case where an actual Nyquist frequency can be determined and is used (rare in practice unless samples are strictly evenly spaced).

PR #416 provides a physical normalization for the amplitude spectrum.  I will replace the power spectral density normalization implementation to follow suit.

#### Environment:
- platform (e.g. Linux, OSX, Windows): OSX
- lightkurve version (e.g. 1.0b6): 1.0b30
- installation method (e.g. pip, conda, source): pip

**Comment by barentsen** [comment]
Thank you for this extremely clear example and explanation, @keatonb!

As a user I would definitely be surprised to see the normalization change merely by adjusting `min_frequency` and `max_frequency`, so I think it makes sense to fix this!  (@christinahedges & @ojhall94 do you agree?)

(As an aside: last week we renamed 'min/max_frequency' into 'minimum/maximum_frequency' to be consistent with AstroPy and the other periodograms, cf. #444.  It won't break anything because we will still support 'min/max_frequency' for quite a while, but it will start raising a *deprecation warning* if you install last week's Lightkurve release.)

**Comment by ojhall94** [comment]
Hi @keatonb, thanks so much for finding this bug (oops)! Minimum/maximum frequency range setting was always intended to slice the periodogram. Glad somebody else is checking on my practices.

**Comment by keatonb** [comment]
Fixed by #491
